contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -2625,15 +2626,19 @@ rename_update_bbf_catalog(RenameStmt *stmt)
 	switch (stmt->renameType)
```

Comment:
Please add some comments regarding:
1. Which catalogs are being modified?
2. Why some of the objects are being modified while others are not? (for both the catalogs)

Reply:
Added a comment.